assets/scss/components/stencil/optimizedCheckout/_optimizedCheckout.scss

```diff
@@ -0,0 +1,104 @@
+// =============================================================================
+// Stencil Checkout  - Customize the Optimized Single-Page Checkout experience
+// =============================================================================
+
+// Header
+.checkoutHeader {
+    background-image: url(stencilImage('optimizedCheckout_custom_background_image', 'optimizedCheckout_custom_background_image_size'));
+    background-size: cover;
+    background-color: stencilColor("optimizedCheckout_header_backgroundColor");
+
+    // Desktop Only
+    @include breakpoint("medium") {
+        text-align: stencilString("optimizedCheckout_logo_position");
+    }
+}
+
+// Order Summary, Mobile Order Summary, Modal Order Summary
+.cart,
+.cartDrawer,
+.cart-modal-body,
+.cart-modal-header {
+    background-color: stencilColor("optimizedCheckout_cart_backgroundColor");
+}
+
+// Headings
+h1,
```

Comment:
I think this file should be split into smaller ones. Also, move `optimizedCheckout` out of `stencil` folder and treat it as a separate unit. For example, you could have something like this:

```
/optimizedCheckout
    /buttons
    /cart
    /forms
    /type
```

What do you think @bc-miko-ademagic?